src/Clients/CaradhrasCardClient.php

```diff
@@ -0,0 +1,484 @@
+<?php
+
+namespace Idez\Caradhras\Clients;
+
+use Idez\Caradhras\Data\Card;
+use Idez\Caradhras\Data\CardCollection;
+use Idez\Caradhras\Data\CardDetails;
+use Idez\Caradhras\Data\CardLimit;
+use Idez\Caradhras\Data\CardMccGroupControl;
+use Idez\Caradhras\Data\CardSettings;
+use Idez\Caradhras\Data\Individual;
+use Idez\Caradhras\Enums\Cards\CardStatus;
+use Idez\Caradhras\Exceptions\CaradhrasException;
+use Idez\Caradhras\Exceptions\CVVMismatchException;
+use Idez\Caradhras\Exceptions\FailedRequestCardBatchException;
+use Idez\Caradhras\Exceptions\FindCardsException;
+use Idez\Caradhras\Exceptions\GetCardDetailsException;
+use Idez\Caradhras\Exceptions\IssuePhysicalCardException;
+
+class CaradhrasCardClient extends BaseApiClient
+{
+    /**
+     * Set card password.
+     *
+     * @param  string  $cardId
+     * @param  string  $password
+     * @return object
+     * @throws \Illuminate\Http\Client\RequestException
+     */
+    public function setCardPassword(string $cardId): object
+    {
+        return $this->apiClient()
+            ->withHeaders(['senha' => str_pad(random_int(1000, 9999), 4, '0', STR_PAD_LEFT)])
```

Comment:
Acredito que não devemos ter essa lógica de senha gerado aleatoriamente dentro do pacote, isso deve ser alterado na chamada desse método dessa API e lá sim aleatorizar esta senha

```suggestion
    public function setCardPassword(string $cardId, $password): object
    {
        return $this->apiClient()
            ->withHeaders(['senha' => str_pad($password, 4, '0', STR_PAD_LEFT)])
```

Reply:
pode crer. entendi agora